src/recon.rs

```diff
@@ -1584,6 +1589,15 @@ unsafe fn decode_coefs<BD: BitDepth>(
     return eob;
 }
 
+#[derive(Clone, Copy)]
+enum CfSelect {
+    // Use `f.frame_thread.cf` at the specified offset.
+    Frame(usize),
+
+    // Use `t.cf`.
+    Task,
+}
```

Comment:
I don't want to nitpick much, so no change necessary now, but in the future, I think it'd be better to make these things doc comments.